src/classes/CRLP_RollupContact_TEST.cls

```diff
@@ -238,7 +238,8 @@ private class CRLP_RollupContact_TEST {
                 Rollups_Contact_SkewMode_Batch_Size__c = 200,
                 Rollups_Contact_Soft_Credit_Batch_Size__c = 200,
                 Rollups_Account_Batch_Size__c = 20,
-                Rollups_Contact_Batch_Size__c = 20
+                Rollups_Contact_Batch_Size__c = 20,
+                Disable_Related_Records_Filter__c = tt == TestType.TestBatch ? true : false
```

Comment:
Need to specifically set Disable Related Records Filter to true in the TestBatch batch type in order for the code block on line 499 of this class to pass